report_pre_thesis/src/sections/moc.tex

```diff
@@ -44,10 +44,49 @@ \subsubsection{Multivariate Oxide Composition Derivation}\label{sec:moc_derivati
 
 \subsubsection{Outlier Removal}\label{sec:outlier_removal}
 
-In their analysis, \citet{andersonImprovedAccuracyQuantitative2017} employed a methodical outlier removal process to enhance model accuracy in multivariate regression. To detect outliers, they utilized influence plots, leveraging statistical measures that reflect each data point's deviation from the model's predictions and their influence on the model due to their position in the predictor space.
+In their analysis, \citet{andersonImprovedAccuracyQuantitative2017} employed a methodical outlier removal process to enhance model accuracy in multivariate regression. To detect outliers, they utilized influence plots, employing statistical measures that reflect each data point's deviation from the model's predictions and their influence on the model due to their position in the predictor space. 
+In this context, the deviation, or the error, are known as residuals and the influence a data point has on the model is known as leverage. By applying the following equation to the latent variables a vector of observation leverages (\(h_t\)) can be computed:
```

Comment:
- $t$ is not defined (it's the PLS scores)
- `to the latent variables a vector of observation leverages ... can be computed` - sentence needs some restructuring or punctuation